sdk/python/feast/cli.py

```diff
@@ -1018,5 +1022,114 @@ def permission_describe(ctx: click.Context, name: str):
     )
 
 
+@feast_permissions_cmd.command(name="check")
+@click.pass_context
+def feast_permissions_check_command(ctx: click.Context):
+    """
+    Validate the permissions configuration
+    """
+    from tabulate import tabulate
+
+    all_unsecured_table: list[Any] = []
+    store = create_feature_store(ctx)
+    permissions = store.list_permissions()
+    objects = cli_utils.fetch_all_feast_objects(
+        store=store,
+    )
+
+    print(
+        f"{Style.BRIGHT + Fore.RED}The following resources are not secured by any permission configuration:{Style.RESET_ALL}"
+    )
+    for o in objects:
+        cli_utils.handle_permissions_check_command(
+            object=o, permissions=permissions, table=all_unsecured_table
+        )
+    print(
+        tabulate(
+            all_unsecured_table,
+            headers=[
+                "NAME",
+                "TYPE",
+            ],
+            tablefmt="plain",
+        )
+    )
+
+    all_unsecured_actions_table: list[Any] = []
+    print(
+        f"{Style.BRIGHT + Fore.RED}The following actions are not secured by any permission configuration (Note: this might not be a security concern, depending on the used APIs):{Style.RESET_ALL}"
+    )
+    for o in objects:
+        cli_utils.handle_permissions_check_command_with_actions(
+            object=o, permissions=permissions, table=all_unsecured_actions_table
+        )
+    print(
+        tabulate(
+            all_unsecured_actions_table,
+            headers=[
+                "NAME",
+                "TYPE",
+                "UNSECURED ACTIONS",
+            ],
+            tablefmt="plain",
+        )
+    )
+
+
+@feast_permissions_cmd.command(name="list-roles")
+@click.option(
+    "--verbose",
+    "-v",
+    is_flag=True,
+    help="Print the resources and actions permitted to each configured role",
+)
+@click.pass_context
+def feast_permissions_list_roles_command(ctx: click.Context, verbose: bool):
+    """
+    List all the configured roles
+    """
+    from tabulate import tabulate
+
+    table: list[Any] = []
+    store = create_feature_store(ctx)
+    permissions = store.list_permissions()
+    if not verbose:
```

Comment:
```suggestion
    if verbose:
```
better because we have else command.

Reply:
I adopted this style because 1) it's the same as the permissions list function, and 2) it shows a sort of top-down priority order, because the non-verbose mode should be the default logic unless we set the optional flag.
But if you really want to change it, I will also update the list function.

Reply:
no big deal.